Samples/3.DebugLog/Program.cs

```diff
@@ -76,6 +76,8 @@ static void Main( string[] args )
 
             // create our steamclient instance
             steamClient = new SteamClient();
+            steamClient.DebugNetworkListener = new NetHookNetworkListener();
```

Comment:
Is this sample really the place for this? Users might have their passwords stored locally in dumps and not realise it.